Probesys/bileto
# dev: Replace Vite by esbuild

## Related issue(s)



N/A

## How to test manually



- Run the application in dev mode (`make docker-start`)
- Verify that the assets (CSS / JS) load correctly
- Run `rm -rf public/assets && ./docker/bin/npm run build`
- Change `dev_assets` by `assets` in `config/services.yaml`
- Verify that the assets (CSS / JS) still load correctly

## Checklist
